zstacklib/zstacklib/utils/sanlock.py

```
@@ -219,15 +266,86 @@ def wait(_):
         direct_init_resource("{}:{}:/dev/mapper/{}-lvmlock:{}".format(lck.lockspace_name, lck.resource_name, lck.vg_name, lck.offset))
 
 
-class Resource(object):
-    def __init__(self, lines, host_id=None, align_size=SMALL_ALIGN_SIZE):
+class DeltaLease(object):
```

Pylint (3.3.7)

[refactor] Class 'DeltaLease' inherits from object, can be safely removed from bases in python3

(R0205)